src/gcp_scanner/scanner.py

```diff
@@ -180,8 +180,10 @@ def get_crawl(
   Returns:
     scan_result: a dictionary with scanning results
   """
-
-  res = crawler.crawl(project_id, client, crawler_config)
+  if crawler.crawler_config:
```

Comment:
You are not actually calling the function here. This statement will always return true.

Reply:
If you are referencing about the crawler_config not having parentheses denoting a function, it is because the crawler_config method is a property, and properties in Python are typically accessed without parentheses.

![Screenshot 2023-11-23 at 6 01 29 PM](https://github.com/google/gcp_scanner/assets/79361941/d1e359d2-c6b7-4377-9ab1-718337c5b41c)

The above testing image shows that, when the scanner is run, it is not always giving true for crawler_config. False for every class except of `StorageBucketsCrawler` (As for now only that class needs the config file).


Please let me know, if I have misunderstood your comment or something else was wrong.

Reply:
Oh my bad, right. NVM. I'd rename the function to make it more clear of what is property about. Right now we have too many `crawler_config`s in the code.

Reply:
Yes, fair point. I have made the changes, changed the name to `has_config_dependency` from `crawler_config`

Reply:
> Yes, fair point. I have made the changes, changed the name to `has_config_dependency` from `crawler_config`

Pushed with the changes.